content/manager/implement-multi-tenancy.md

```diff
@@ -0,0 +1,105 @@
+
+##Multi-Tenancy for Premium Users##
+There are two options for using the multi-tenancy function in Cloudify Manager, as a standalone function that is not connected to an existing user managment system (LDAP/AD) or as a function that is integrated with the LDAP/AD user definitions. You must select one option only. If you already have LDAP/AD running in your organization, you probably want to use that, to streamline the process. If you do not use the LDAP/AD option, you must define each user individually.
+
+In the following diagram note that there are three tenants in Cloudify Manager. Tenants can be assigned groups of users, individual users, or a combination of both groups and individuals.
+ 
+![Multi-tenancy Options]({{< img "manager/multi-tenancy-options.png" >}})
+
+###Basic Workflow for Adding Tenants, User Groups and Individual Users###
+
+This section describes a basic workflow for adding a tenant, then adding one or more users to it. It is relevant only to Enterprise users.
+1. You must start with a tenant. When you install Cloudify, a default tenant is also installed. Either use the default tenant or create one or more new tenants.  
+  Use the tenant-related commands that are listed later in this topic to create a new tenant.
+2. Each tenant has a default Admin user that cannot be deleted. You can add other users to the tenant.  
+  Use the user-related commands that are listed later in this topic to add a new user or user group.
+  You can add users individually or as part of a group. You can add them manually, or via your organization's LDAP/AD setup.  
+  1. To add a user or user group via LDAP/AD, connect to the LDAP/AD service.
+  2. To add a single user or user group manually, specify the name of the user or group as part of the command.
+  3. Specify a password and role for users that you add manually.  
+    Users who are added from LDAP/AD retain their LDAP/AD password. You might still want to assign them a role.
+
+####Managing Tenants####
+
+Tenants can be added or deleted by an `administrator` user.  
+
+When a tenant is created, a default `administrator` is created for it. Additional users and user groups can also be added. Those users can then deploy blueprints on that tenant.
+
+When a user runs the `list` command on a tenant, they see all the tenants to which they are assigned - both through individual assignment and through membership of an assigned group.
+
+
+#####Deleting Tenants#####
+
+You must ensure that any tenant that you want to delete is empty, meaning that it does not contain any resources.
+
+The default tenant that is created during the Cloudify Manager installation/bootstrapping and its `administrator` cannot be deleted.
+
+Using the API, an Administrator user can delete a resource on a tenant other than the one which they are currently managing.
```

Comment:
I think we first need to say that all commands are in the context of a tenant, but admins can still refer to tenants other than the one they are currently in the context of. otherwise this sentence won't be clear.